alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineCliParserTest.scala

```diff
@@ -20,6 +20,34 @@ import org.scalatest.FunSuite
 
 class AlertEngineCliParserTest extends FunSuite {
 
+  /*
+  *function to set environmental variables for testing
+   */
+  def setEnv(key: String, value: String) = {
+
+    val field = System.getenv().getClass.getDeclaredField("m")
+    field.setAccessible(true)
+    val map = field.get(System.getenv()).asInstanceOf[java.util.Map[java.lang.String, java.lang.String]]
+    map.put(key, value)
+
+  }
+
+  test("Test Env variable setup") {
```

Comment:
Probably should add a comment to this one saying it only tests test framework code